.github/workflows/playwright-cron.yml

```diff
@@ -0,0 +1,30 @@
+name: Playwright Tests
+on:
+  workflow_dispatch:  
+  schedule:
+    # every hour, Mon - Fri
+    - cron: "0 * * * 1-5"
```

Comment:
How about only 9-5?

Reply:
I like this from a "You shouldn't be working outside of hours" perspective - but do you think we should be optimising for compute time? I'm not 100% sure we pay for it (although burning event a tiny amount of power for nothing seems silly).

It's also nice we can call this manually via `workflow_dispatch`. As a secondary PR - I will have a think about how we socialise this stuff.